Unit tests are falling

Unit tests are currently falling on `internal/auth/auth.go`  :

```
% make clean;make all
[...]
go test github.com/fabric8-services/fabric8-jenkins-proxy/cmd/fabric8-jenkins-proxy github.com/fabric8-services/fabric8-jenkins-proxy/cmd/osio github.com/fabric8-services/fabric8-jenkins-proxy/cmd/osio/cmd github.com/fabric8-services/fabric8-jenkins-proxy/internal/api github.com/fabric8-services/fabric8-jenkins-proxy/internal/auth github.com/fabric8-services/fabric8-jenkins-proxy/internal/clients github.com/fabric8-services/fabric8-jenkins-proxy/internal/configuration github.com/fabric8-services/fabric8-jenkins-proxy/internal/jenkinsapi github.com/fabric8-services/fabric8-jenkins-proxy/internal/metric github.com/fabric8-services/fabric8-jenkins-proxy/internal/proxy github.com/fabric8-services/fabric8-jenkins-proxy/internal/proxy/reverseproxy github.com/fabric8-services/fabric8-jenkins-proxy/internal/router github.com/fabric8-services/fabric8-jenkins-proxy/internal/storage github.com/fabric8-services/fabric8-jenkins-proxy/internal/testutils github.com/fabric8-services/fabric8-jenkins-proxy/internal/testutils/mock github.com/fabric8-services/fabric8-jenkins-proxy/internal/util github.com/fabric8-services/fabric8-jenkins-proxy/internal/util/logging github.com/fabric8-services/fabric8-jenkins-proxy/internal/version
internal/auth/auth.go:189: Entry.Info call has possible formatting directive %s
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/cmd/fabric8-jenkins-proxy	(cached)
?   	github.com/fabric8-services/fabric8-jenkins-proxy/cmd/osio	[no test files]
?   	github.com/fabric8-services/fabric8-jenkins-proxy/cmd/osio/cmd	[no test files]
?   	github.com/fabric8-services/fabric8-jenkins-proxy/internal/api	[no test files]
FAIL	github.com/fabric8-services/fabric8-jenkins-proxy/internal/auth [build failed]
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/clients	(cached)
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/configuration	(cached)
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/jenkinsapi	(cached)
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/metric	(cached)
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/proxy	(cached)
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/proxy/reverseproxy	(cached)
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/router	(cached)
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/storage	(cached)
?   	github.com/fabric8-services/fabric8-jenkins-proxy/internal/testutils	[no test files]
?   	github.com/fabric8-services/fabric8-jenkins-proxy/internal/testutils/mock	[no test files]
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/util	(cached)
ok  	github.com/fabric8-services/fabric8-jenkins-proxy/internal/util/logging	(cached)
?   	github.com/fabric8-services/fabric8-jenkins-proxy/internal/version	[no test files]
make: *** [test] Error 2
```

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -3,518 +3,311 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:6978a38432a017763a148afbc7ce6491734b54292af7d3e969d84d2e9dd242e2"
   name = "github.com/Azure/go-ansiterm"
-  packages = [
-    ".",
-    "winterm",
-  ]
-  pruneopts = ""
+  packages = [".","winterm"]
   revision = "d6e3b3328b783f23731bc4d058875b0371ff8109"
 
 [[projects]]
-  digest = "1:e43b98837e21e7f9ef552445e1fcfa92e36bfc0a3e0bd78f2a5e0bef7a4378e3"
   name = "github.com/Microsoft/go-winio"
   packages = ["."]
-  pruneopts = ""
   revision = "7da180ee92d8bd8bb8c37fc560e673e6557c392f"
   version = "v0.4.7"
 
 [[projects]]
   branch = "master"
-  digest = "1:3721a10686511b80c052323423f0de17a8c06d417dbdd3b392b1578432a33aae"
   name = "github.com/Nvveen/Gotty"
   packages = ["."]
-  pruneopts = ""
   revision = "cd527374f1e5bff4938207604a14f2e38a9cf512"
 
 [[projects]]
   branch = "master"
-  digest = "1:fca298802a2ab834d6eb0e284788ae037ebc324c0f325ff92c5eea592d189cc5"
   name = "github.com/beorn7/perks"
   packages = ["quantile"]
-  pruneopts = ""
   revision = "3a771d992973f24aa725d07868b467d1ddfceafb"
 
 [[projects]]
-  digest = "1:1cf61f0228e64d1bfddaa42bfd1f5047ed3626925b9bea69bbcb9168472dae23"
   name = "github.com/cenk/backoff"
   packages = ["."]
-  pruneopts = ""
   revision = "61153c768f31ee5f130071d08fc82b85208528de"
   version = "v1.1.0"
 
 [[projects]]
   branch = "master"
-  digest = "1:a93bb454acdb148dece332f89a22564b4fa14873a5392e117aa150a022d8e9f9"
   name = "github.com/containerd/continuity"
   packages = ["pathdriver"]
-  pruneopts = ""
   revision = "b2b946a77f5973f420514090d6f6dd58b08303f0"
 
 [[projects]]
-  digest = "1:0a39ec8bf5629610a4bc7873a92039ee509246da3cef1a0ea60f1ed7e5f9cea5"
   name = "github.com/davecgh/go-spew"
   packages = ["spew"]
-  pruneopts = ""
   revision = "346938d642f2ec3594ed81d874461961cd0faa76"
   version = "v1.1.0"
 
 [[projects]]
-  digest = "1:2dff93b5f7538cd4b00e15bec03c6b9ae2c17628f3c1b2b7dbeb617a51de7aa6"
   name = "github.com/dgrijalva/jwt-go"
   packages = ["."]
-  pruneopts = ""
   revision = "dbeaa9332f19a944acb5736b4456cfcc02140e29"
   version = "v3.1.0"
 
 [[projects]]
-  digest = "1:97a54e3ae9d27f456432ff12c8e4287036705a94f08b24a4514e9698a9653929"
   name = "github.com/docker/docker"
-  packages = [
-    "api/types",
-    "api/types/blkiodev",
-    "api/types/container",
-    "api/types/filters",
-    "api/types/mount",
-    "api/types/network",
-    "api/types/registry",
-    "api/types/strslice",
-    "api/types/swarm",
-    "api/types/swarm/runtime",
-    "api/types/versions",
-    "opts",
-    "pkg/archive",
-    "pkg/fileutils",
-    "pkg/homedir",
-    "pkg/idtools",
-    "pkg/ioutils",
-    "pkg/jsonmessage",
-    "pkg/longpath",
-    "pkg/mount",
-    "pkg/pools",
-    "pkg/stdcopy",
-    "pkg/system",
-    "pkg/term",
-    "pkg/term/windows",
-  ]
-  pruneopts = ""
+  packages = ["api/types","api/types/blkiodev","api/types/container","api/types/filters","api/types/mount","api/types/network","api/types/registry","api/types/strslice","api/types/swarm","api/types/swarm/runtime","api/types/versions","opts","pkg/archive","pkg/fileutils","pkg/homedir","pkg/idtools","pkg/ioutils","pkg/jsonmessage","pkg/longpath","pkg/mount","pkg/pools","pkg/stdcopy","pkg/system","pkg/term","pkg/term/windows"]
   revision = "fe8aac6f5ae413a967adb0adad0b54abdfb825c4"
 
 [[projects]]
-  digest = "1:5b20afc76a36d3994194e2612e83b51bc2b12db3d4d2a722b24474b2d0e3a890"
   name = "github.com/docker/go-connections"
   packages = ["nat"]
-  pruneopts = ""
   revision = "3ede32e2033de7505e6500d6c868c2b9ed9f169d"
   version = "v0.3.0"
 
 [[projects]]
-  digest = "1:a406cae5eda48c01f8171bd47beb038751393f25ac06774ce04f9d6b0b703f17"
   name = "github.com/docker/go-units"
   packages = ["."]
-  pruneopts = ""
   revision = "0dadbb0345b35ec7ef35e228dabb8de89a65bf52"
   version = "v0.3.2"
 
 [[projects]]
-  digest = "1:b2106f1668ea5efc1ecc480f7e922a093adb9563fd9ce58585292871f0d0f229"
   name = "github.com/fsnotify/fsnotify"
   packages = ["."]
-  pruneopts = ""
   revision = "c2828203cd70a50dcccfb2761f8b1f8ceef9a8e9"
   version = "v1.4.7"
 
 [[projects]]
-  digest = "1:83d75b7e51d7580810a7dc8bb83b9ed43fa16314af4de023d2cb5ff2d76578ea"
   name = "github.com/fsouza/go-dockerclient"
   packages = ["."]
-  pruneopts = ""
   revision = "2ff310040c161b75fa19fb9b287a90a6e03c0012"
   version = "1.1"
 
 [[projects]]
-  digest = "1:a24a87dc28e137946d37f585b2f5a955e24e22fdb4cc3244167c86c1bb7d5c65"
   name = "github.com/gogo/protobuf"
   packages = ["proto"]
-  pruneopts = ""
   revision = "342cbe0a04158f6dcb03ca0079991a51a4248c02"
   version = "v0.5"
 
 [[projects]]
-  digest = "1:9727058af2b19618488a3db777944ef9873252e974707cba43583ab3517b28f6"
   name = "github.com/golang/protobuf"
   packages = ["proto"]
-  pruneopts = ""
   revision = "925541529c1fa6821df4e44ce2723319eb2be768"
   version = "v1.0.0"
 
 [[projects]]
   branch = "master"
-  digest = "1:648d4a05fa2648a1cea97132b077aa15e7b057cad78b34bdf998ab5e4a64ee52"
   name = "github.com/hashicorp/hcl"
-  packages = [
-    ".",
-    "hcl/ast",
-    "hcl/parser",
-    "hcl/scanner",
-    "hcl/strconv",
-    "hcl/token",
-    "json/parser",
-    "json/scanner",
-    "json/token",
-  ]
-  pruneopts = ""
+  packages = [".","hcl/ast","hcl/parser","hcl/scanner","hcl/strconv","hcl/token","json/parser","json/scanner","json/token"]
   revision = "23c074d0eceb2b8a5bfdbb271ab780cde70f05a8"
 
 [[projects]]
-  digest = "1:870d441fe217b8e689d7949fef6e43efbc787e50f200cb1e70dbca9204a1d6be"
   name = "github.com/inconshreveable/mousetrap"
   packages = ["."]
-  pruneopts = ""
   revision = "76626ae9c91c4f2a10f34cad8ce83ea42c93bb75"
   version = "v1.0"
 
 [[projects]]
-  digest = "1:8f2a3111f21ba5482d99b7d6f7d98af7f233f18ebfdbfd6eedc475371177d75d"
   name = "github.com/jinzhu/gorm"
   packages = ["."]
-  pruneopts = ""
   revision = "5174cc5c242a728b435ea2be8a2f7f998e15429b"
   version = "v1.0"
 
 [[projects]]
   branch = "master"
-  digest = "1:a0c0f06f12f4b83b89eec79114265c54e9e300b8ddda0f0f3fb49a685283fa6c"
   name = "github.com/jinzhu/inflection"
   packages = ["."]
-  pruneopts = ""
   revision = "1c35d901db3da928c72a72d8458480cc9ade058f"
 
 [[projects]]
-  digest = "1:3c818dada3e41bdb0f509f78e6775610f1bb179449ec8c4c86a45fae35460f3f"
   name = "github.com/julienschmidt/httprouter"
   packages = ["."]
-  pruneopts = ""
   revision = "8c199fb6259ffc1af525cc3ad52ee60ba8359669"
   version = "v1.1"
 
 [[projects]]
-  digest = "1:ada2b83c3f1a382f0bf2d8b56a0e2f39f324a5c4a20eb1b3574697fa2ae67c1a"
   name = "github.com/lib/pq"
-  packages = [
-    ".",
-    "oid",
-  ]
-  pruneopts = ""
+  packages = [".","oid"]
   revision = "83612a56d3dd153a94a629cd64925371c9adad78"
 
 [[projects]]
-  digest = "1:fd4be92c5f012c472da2080e22deb0951cff42a489a6eb5d200e37dbcfeab28f"
   name = "github.com/magiconair/properties"
   packages = ["."]
-  pruneopts = ""
   revision = "d419a98cdbed11a922bf76f257b7c4be79b50e73"
   version = "v1.7.4"
 
 [[projects]]
   branch = "master"
-  digest = "1:22adea92fb070dc6dd0bebeeb578072b3f4692087e4f5c893da7d27635d61f82"
   name = "github.com/matryer/resync"
   packages = ["."]
-  pruneopts = ""
   revision = "d39c09a11215c84aab0b65e323fc47dd6e276af1"
 
 [[projects]]
-  digest = "1:da0ae6b7e9dee808befe2fd96fba4b6b607a2dbb744e13fc7aae48a592edb06d"
   name = "github.com/matttproud/golang_protobuf_extensions"
   packages = ["pbutil"]
-  pruneopts = ""
   revision = "3247c84500bff8d9fb6d579d800f20b3e091582c"
   version = "v1.0.0"
 
 [[projects]]
   branch = "master"
-  digest = "1:a31c8ff06bf8ccc1826e3a28a4132bf217a53f3b096e6a03dcac7b6a9eb796f9"
   name = "github.com/mitchellh/mapstructure"
   packages = ["."]
-  pruneopts = ""
   revision = "b4575eea38cca1123ec2dc90c26529b5c5acfcff"
 
 [[projects]]
-  digest = "1:5d9b668b0b4581a978f07e7d2e3314af18eb27b3fb5d19b70185b7c575723d11"
   name = "github.com/opencontainers/go-digest"
   packages = ["."]
-  pruneopts = ""
   revision = "279bed98673dd5bef374d3b6e4b09e2af76183bf"
   version = "v1.0.0-rc1"
 
 [[projects]]
-  digest = "1:0d08f7224705b1df80beee92ffbdc63ab13fd6f6eb80bf287735f9bc7e8b83eb"
   name = "github.com/opencontainers/image-spec"
-  packages = [
-    "specs-go",
-    "specs-go/v1",
-  ]
-  pruneopts = ""
+  packages = ["specs-go","specs-go/v1"]
   revision = "d60099175f88c47cd379c4738d158884749ed235"
   version = "v1.0.1"
 
 [[projects]]
-  digest = "1:2b274e4011f3dea29d0cd91e23e0f977fa02470099c827808e9dedeb75f36b9c"
   name = "github.com/opencontainers/runc"
-  packages = [
-    "libcontainer/system",
-    "libcontainer/user",
-  ]
-  pruneopts = ""
+  packages = ["libcontainer/system","libcontainer/user"]
   revision = "baf6536d6259209c3edfa2b22237af82942d3dfa"
   version = "v0.1.1"
 
 [[projects]]
-  digest = "1:4c0404dc03d974acd5fcd8b8d3ce687b13bd169db032b89275e8b9d77b98ce8c"
   name = "github.com/patrickmn/go-cache"
   packages = ["."]
-  pruneopts = ""
   revision = "a3647f8e31d79543b2d0f0ae2fe5c379d72cedc0"
   version = "v2.1.0"
 
 [[projects]]
-  digest = "1:fef37519c971ab3c464318a0c9da6a7fb9c21439aa587e61bf10af651e6119b9"
   name = "github.com/pelletier/go-toml"
   packages = ["."]
-  pruneopts = ""
   revision = "acdc4509485b587f5e675510c4f2c63e90ff68a8"
   version = "v1.1.0"
 
 [[projects]]
-  digest = "1:7365acd48986e205ccb8652cc746f09c8b7876030d53710ea6ef7d0bd0dcd7ca"
   name = "github.com/pkg/errors"
   packages = ["."]
-  pruneopts = ""
   revision = "645ef00459ed84a119197bfb8d8205042c6df63d"
   version = "v0.8.0"
 
 [[projects]]
-  digest = "1:256484dbbcd271f9ecebc6795b2df8cad4c458dd0f5fd82a8c2fa0c29f233411"
   name = "github.com/pmezard/go-difflib"
   packages = ["difflib"]
-  pruneopts = ""
   revision = "792786c7400a136282c1664665ae0a8db921c6c2"
   version = "v1.0.0"
 
 [[projects]]
-  digest = "1:981835985f655d1d380cc6aa7d9fa9ad7abfaf40c75da200fd40d864cd05a7c3"
   name = "github.com/prometheus/client_golang"
-  packages = [
-    "prometheus",
-    "prometheus/promhttp",
-  ]
-  pruneopts = ""
+  packages = ["prometheus","prometheus/promhttp"]
   revision = "c5b7fccd204277076155f10851dad72b76a49317"
   version = "v0.8.0"
 
 [[projects]]
   branch = "master"
-  digest = "1:83bf37d060fca77e959fe5ceee81e58bbd1b01836f4addc70043a948e9912547"
   name = "github.com/prometheus/client_model"
   packages = ["go"]
-  pruneopts = ""
   revision = "99fa1f4be8e564e8a6b613da7fa6f46c9edafc6c"
 
 [[projects]]
   branch = "master"
-  digest = "1:5e349d7330e5172c9d21e3ece41fb07e8d0510e3efdbcc554c7db32036456627"
   name = "github.com/prometheus/common"
-  packages = [
-    "expfmt",
-    "internal/bitbucket.org/ww/goautoneg",
-    "model",
-  ]
-  pruneopts = ""
+  packages = ["expfmt","internal/bitbucket.org/ww/goautoneg","model"]
   revision = "38c53a9f4bfcd932d1b00bfc65e256a7fba6b37a"
 
 [[projects]]
   branch = "master"
-  digest = "1:bb59cf3b7ba604c10d35ba04d3246434d1e6adf821688e4cd33d3f4090a8b62a"
   name = "github.com/prometheus/procfs"
-  packages = [
-    ".",
-    "internal/util",
-    "nfs",
-    "xfs",
-  ]
-  pruneopts = ""
+  packages = [".","internal/util","nfs","xfs"]
   revision = "780932d4fbbe0e69b84c34c20f5c8d0981e109ea"
 
 [[projects]]
-  digest = "1:009cc9ec2bee568eb427f9471d386f4ed9489086cd0160e4c90a015fbe9db387"
   name = "github.com/rs/cors"
   packages = ["."]
-  pruneopts = ""
   revision = "feef513b9575b32f84bafa580aad89b011259019"
   version = "v1.3.0"
 
 [[projects]]
-  digest = "1:6b55df4b0517a459af9d3879c99330af4367adcf45f3d0d37ded80a6272ae057"
   name = "github.com/satori/go.uuid"
   packages = ["."]
-  pruneopts = ""
   revision = "879c5887cd475cd7864858769793b2ceb0d44feb"
   version = "v1.1.0"
 
 [[projects]]
-  digest = "1:7e30913ef139d0a90b81742683d63ce8f89e35d234323e32ae3524a7fe89bc03"
   name = "github.com/sirupsen/logrus"
-  packages = [
-    ".",
-    "hooks/test",
-  ]
-  pruneopts = ""
+  packages = [".","hooks/test"]
   revision = "202f25545ea4cf9b191ff7f846df5d87c9382c2b"
   version = "v1.0.0"
 
 [[projects]]
-  digest = "1:a58ad73ed272ec394398449105271c425ac634f908bda45825c466abb948ca86"
   name = "github.com/spf13/afero"
-  packages = [
-    ".",
-    "mem",
-  ]
-  pruneopts = ""
+  packages = [".","mem"]
   revision = "bb8f1927f2a9d3ab41c9340aa034f6b803f4359c"
   version = "v1.0.2"
 
 [[projects]]
-  digest = "1:6ff9b74bfea2625f805edec59395dc37e4a06458dd3c14e3372337e3d35a2ed6"
   name = "github.com/spf13/cast"
   packages = ["."]
-  pruneopts = ""
   revision = "acbeb36b902d72a7a4c18e8f3241075e7ab763e4"
   version = "v1.1.0"
 
 [[projects]]
-  digest = "1:101444b6da4e706fc5d1b997a39c982ecc3eaede505ae0888142991266eb499b"
   name = "github.com/spf13/cobra"
   packages = ["."]
-  pruneopts = ""
   revision = "7b2c5ac9fc04fc5efafb60700713d4fa609b777b"
   version = "v0.0.1"
 
 [[projects]]
   branch = "master"
-  digest = "1:104517520aab91164020ab6524a5d6b7cafc641b2e42ac6236f6ac1deac4f66a"
   name = "github.com/spf13/jwalterweatherman"
   packages = ["."]
-  pruneopts = ""
   revision = "7c0cea34c8ece3fbeb2b27ab9b59511d360fb394"
 
 [[projects]]
-  digest = "1:261bc565833ef4f02121450d74eb88d5ae4bd74bfe5d0e862cddb8550ec35000"
   name = "github.com/spf13/pflag"
   packages = ["."]
-  pruneopts = ""
   revision = "e57e3eeb33f795204c1ca35f56c44f83227c6e66"
   version = "v1.0.0"
 
 [[projects]]
-  digest = "1:ff456b6c80595ee3080e46eb9a7dffa826df57de2d678d590027455d37e11213"
   name = "github.com/spf13/viper"
   packages = ["."]
-  pruneopts = ""
   revision = "25b30aa063fc18e48662b86996252eabdcf2f0c7"
   version = "v1.0.0"
 
 [[projects]]
-  digest = "1:8e7d1910593db3b38b0c3e4bf3f82bc041dfddfed5f6d36a47f3ad7e72710d8d"
   name = "github.com/stretchr/testify"
-  packages = [
-    "assert",
-    "require",
-  ]
-  pruneopts = ""
+  packages = ["assert","require"]
   revision = "69483b4bd14f5845b5a1e55bca19e954e827f1d0"
   version = "v1.1.4"
 
 [[projects]]
   branch = "master"
-  digest = "1:e2b7b8d650c35d98a0d94d9ae33c5bdd9043eb84c1ac74baaca5804c61e8c7ad"
   name = "golang.org/x/net"
-  packages = [
-    "context",
-    "context/ctxhttp",
-    "html",
-    "html/atom",
-  ]
-  pruneopts = ""
+  packages = ["context","context/ctxhttp","html","html/atom"]
   revision = "5ccada7d0a7ba9aeb5d3aca8d3501b4c2a509fec"
 
 [[projects]]
   branch = "master"
-  digest = "1:b23045131cd8f98c7b805c13c2e0e8226a0e3b93737ac0d24a4dbe8fd09a71cb"
   name = "golang.org/x/sys"
-  packages = [
-    "unix",
-    "windows",
-  ]
-  pruneopts = ""
+  packages = ["unix","windows"]
   revision = "af50095a40f9041b3b38960738837185c26e9419"
 
 [[projects]]
   branch = "master"
-  digest = "1:e390b35ff3921303ce8a400363d6fa5174f24ff5fe01cbb98a8f8b3b7f84b1da"
   name = "golang.org/x/text"
-  packages = [
-    "internal/gen",
-    "internal/triegen",
-    "internal/ucd",
-    "transform",
-    "unicode/cldr",
-    "unicode/norm",
-  ]
-  pruneopts = ""
+  packages = ["internal/gen","internal/triegen","internal/ucd","transform","unicode/cldr","unicode/norm"]
   revision = "e19ae1496984b1c655b8044a65c0300a3c878dd3"
 
 [[projects]]
-  digest = "1:c46aeb9791abe1b53a9173720dfaf174905ff8660be37bdbf926079e5a3cb0d5"
   name = "gopkg.in/ory-am/dockertest.v3"
   packages = ["."]
-  pruneopts = ""
   revision = "59e045640952457da1046932eae73e20cb3afe70"
   version = "v3.1.3"
 
 [[projects]]
   branch = "v2"
-  digest = "1:4b4e5848dfe7f316f95f754df071bebfb40cf4482da62e17e7e1aebdf11f4918"
   name = "gopkg.in/yaml.v2"
   packages = ["."]
-  pruneopts = ""
   revision = "d670f9405373e636a5a2765eea47fac0c9bc91a4"
 
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  input-imports = [
-    "github.com/dgrijalva/jwt-go",
-    "github.com/jinzhu/gorm",
-    "github.com/julienschmidt/httprouter",
-    "github.com/lib/pq",
-    "github.com/matryer/resync",
-    "github.com/patrickmn/go-cache",
-    "github.com/pkg/errors",
-    "github.com/prometheus/client_golang/prometheus",
-    "github.com/prometheus/client_golang/prometheus/promhttp",
-    "github.com/prometheus/client_model/go",
-    "github.com/rs/cors",
-    "github.com/satori/go.uuid",
-    "github.com/sirupsen/logrus",
-    "github.com/sirupsen/logrus/hooks/test",
-    "github.com/spf13/cobra",
-    "github.com/spf13/viper",
-    "github.com/stretchr/testify/assert",
-    "github.com/stretchr/testify/require",
-    "golang.org/x/net/html",
-    "gopkg.in/ory-am/dockertest.v3",
-  ]
+  inputs-digest = "1fdccbea14f287a8a6e97e381ad11328ba335d53b7e25bc3f0ca924f1e452757"
   solver-name = "gps-cdcl"
   solver-version = 1

--- a/internal/auth/auth.go
+++ b/internal/auth/auth.go
@@ -186,7 +186,7 @@ func (c *Client) updatePublicKeysOnce() error {
 func (c *Client) updatePublicKeys() error {
 	tokenURL := strings.TrimRight(c.URL, "/") + "/api/token/keys?format=pem"
 
-	c.log.Info("Fetching public keys from %s", tokenURL)
+	c.log.Infof("Fetching public keys from %s", tokenURL)
 	resp, err := util.HTTPClient().Get(tokenURL)
 	if err != nil {
 		return err

